app/.env

```diff
@@ -0,0 +1 @@
+BO_URL=http://localhost:1337/
```

Comment:
Should be implemented inside .env.local
if the value doesnt exist here, he will search in other .env corresponding of environement (normally)
```suggestion 
BO_URL=
```

Reply:
guess you right i need to check this up